proto/BUILD

```diff
@@ -3,15 +3,15 @@ load("@rules_proto//proto:defs.bzl", "proto_library")
 
 # _____ build cpp from proto _____
 cc_proto_library(
-  name = "spec_cc_proto",
+  name = "cloud_event_cc_proto",
```

Comment:
The spec name is "cloud_events" (not "cloud_event"). Here and everywhere.

Reply:
I take this back. I now see that the CloudEvents proposal is to actually proposing to call the proto CloudEvent.